docs/certificate-notice.md

```diff
@@ -0,0 +1,12 @@
+# Certificate Notice
+
+__IMPORTANT:__
+
+- The content of this repository is not guaranteed to be up to date. It is also not guaranteed to be a complete list of CA certificates used across Azure Clouds or when using Azure Stack, Azure IoT Edge, or Azure Protocol Gateway.
+- Always prefer using the local system's Trusted Root Certificate Authority store instead of hardcoded certificates (e.g. the hardcoded certificates used in our `demo_config.h` files).
+- Azure Root certificates may change with or without prior notice (e.g., if they expire or are revoked). It is important that devices are able to add or remove trusted root certificates.
+- Support for at least two certificates is required to maintain device connectivity during CA certificate changes.
+
+## Additional Information
```

Comment:
We could add the DER information from the comments here together with an example on how a Base64 encoded certificate can be transformed into a DER here.

Reply:
I think that gets in the weeds a little too much. I'll probably leave it as is to not have this readme be an openssl and certificates lesson.